functions/helloworld/storage_generic/hello_storage_generic.go

```diff
@@ -1,80 +0,0 @@
-// Copyright 2019 Google LLC
-//
-// Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-//
-//     https://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
-
-// [START functions_helloworld_storage_generic]
-
-// Package helloworld provides a set of Cloud Functions samples.
-package helloworld
-
-import (
-	"context"
-	"fmt"
-	"log"
-	"time"
-
-	"cloud.google.com/go/functions/metadata"
-)
-
-// GCSEvent is the payload of a GCS event.
-type GCSEvent struct {
```

Comment:
Just confirming: this struct exists somewhere else to be referenced from the storage trigger docs?

Reply:
Yes - I think it exists in `functions/helloworld/hello_cloud_storage.go` (the file this sample was merged into).